src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,8 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Move the 'box-sizing' property to a universal selector (*) to ensure it applies to all elements.